README.md

```diff
@@ -1 +1,194 @@
-# backend_capstone
\ No newline at end of file
+<div align="center">
```

Comment:
- [ ] There are 2 tests that are failing on my end:

![image](https://github.com/skabeo/backend_capstone/assets/57421823/65efe8ea-9fef-436d-910e-d6ee9275f4f6)

Kindly, fix those tests so they pass 👍